enhancements/network/whereabouts-sticky-ips.md

```diff
@@ -0,0 +1,84 @@
+---
+title: whereabouts-sticky-ips
+authors:
+  - "@dougbtv"
+reviewers:
+  - "@s1061123"
+  - "@zshi-redhat"
+approvers:
+  - "@s1061123"
+  - "@zshi-redhat"
+creation-date: 2020-08-05
+last-updated: 2020-08-05
+status: provisional
+---
+
+
+# Whereabouts "Sticky IP addresses"
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This feature implements functionality for Whereabouts where workloads are assigned an IP address, and when those workloads are terminated and restarted, are assigned the same IP address.
```

Comment:
Is this an enhancement to something we already ship and support, or proposing Whereabouts in general?

If it's something we already ship, where could I read more about its inclusion in OpenShift and what use cases it solves today?

If it's introducing Whereabouts as a new thing, can you update this to do two things: first introduce the problem without mention of the solution (whereabouts).  Then can you summarize what wherabouts is, where it comes from, and how people can learn more?

Reply:
whereabouts is something we already ship, but that was pre-enhancements, and I have no idea what it does (although I'm sure Doug explained it to me at some point) so yeah, it would be good to have this enhancement as both an explanation of whereabouts _and_ an explanation of the new 4.7 feature on top of that.

Reply:
+1 to explaining both, thanks :)